pkgs/top-level/all-packages.nix

```diff
@@ -19414,6 +19414,12 @@ in
     };
   };
 
+  spotifyd = callPackage ../applications/audio/spotifyd {
+    withALSA = stdenv.isLinux;
+    withPulseAudio = config.pulseaudio or true;
```

Comment:
this would always evaluate to true.

Does it make sense to build it with both alsa and pulseaudio support on linux, and portaudio on darwin by default (and move that logic into `spotifyd/default.nix`)?

Reply:
```
nix-repl> { pulseaudio = false ; }.pulseaudio or true
false
```

Reply:
right - saw too much python in Februrary :-D